src/hooks/usePrefix.ts

```diff
@@ -1,31 +1,7 @@
 import Path from "path"
+import useConfig from "hooks/useConfig.ts"
 
-const prefix = (() => {
-  //NOTE doesn't work for scripts as Deno.run doesn't push through most env :/
-  const env = Deno.env.get("TEA_PREFIX")
-  if (env) {
-    return new Path(env)
-  } else {
-    // we’re either deno.land/vx/bin/deno, tea.xyz/vx/bin/tea or some symlink to the latter
-    const shelf = new Path(Deno.execPath())
-      .readlink() // resolves the leaf symlink (if any)
-      .parent()
-      .parent()
-      .parent()
-
-    switch (shelf.basename()) {
-    case 'tea.xyz':
-    case 'deno.land':
-      return shelf.parent()
```

Comment:
I wonder if we still need this bit. We migrated to “you must define TEA_PREFIX if you aren't in ~/.tea” kinda thing